src/client/render/render-core.js

```diff
@@ -83,7 +89,7 @@ let renderCore = function() {
 			} else { // Initialize
 				this.mainScene = new THREE.Scene();
 				_renderer = new THREE.WebGLRenderer();
-				_renderer.debug.checkShaderErrors = false;
+				_renderer.debug.checkShaderErrors = true;
```

Comment:
Sort of indirect leftover logging, I think?

Reply:
Ye oops